TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -224,7 +226,8 @@ public static void main(String[] args) throws Exception {
         var activeContextLogger = LoggerFactory.getLogger(ALL_ACTIVE_CONTEXTS_MONITOR_LOGGER);
         var params = parseArgs(args);
         URI uri;
-        System.err.println("Starting Traffic Replayer");
+        final var workerId = UUID.randomUUID().toString();
+        System.err.println("Starting Traffic Replayer with id=" + workerId);
```

Comment:
Why are we doing UUID workerIds here?

Reply:
Seems like we should use the `ProcessHelpers.getNodeInstanceName()` to get the id